backend/src/test/resources/application-default.yaml

```diff
@@ -223,6 +223,13 @@ simple-report-initialization:
       model: LumiraDx SARS-CoV-2 Ag Test*
       loinc-code: "95209-3"
       swab-type: "445297001"
+  supported-diseases:
```

Comment:
Hmm, we might want to hop on a call to work out the differences between this and #3560 - I think this block is duplicating some of the changes there.

Reply:
Happy to! 😄 feel free to grab any spot on my cal!